ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/AcscsEmailIntegrationForm.tsx

```diff
@@ -20,41 +20,35 @@ export type ACSCSEmailIntegration = {
     type: 'acscsEmail';
 } & NotifierIntegrationBase;
 
-export type ACSCSEmailIntegrationFormValues = {
-    notifier: ACSCSEmailIntegration;
-};
+export type ACSCSEmailIntegrationFormValues = ACSCSEmailIntegration;
 
 export const validationSchema = yup.object().shape({
-    notifier: yup.object().shape({
-        name: yup.string().trim().required('Email integration name is required'),
-        labelDefault: yup
-            .string()
-            .trim()
-            .required('A default recipient email address is required')
-            .email('Must be a valid default recipient email address'),
-        labelKey: yup.string(),
-    }),
+    name: yup.string().trim().required('Email integration name is required'),
+    labelDefault: yup
+        .string()
+        .trim()
+        .required('A default recipient email address is required')
+        .email('Must be a valid default recipient email address'),
+    labelKey: yup.string(),
 });
 
 export const defaultValues: ACSCSEmailIntegrationFormValues = {
-    notifier: {
-        id: '',
-        name: '',
-        type: 'acscsEmail',
-        labelDefault: '',
-        labelKey: '',
-        uiEndpoint: window.location.origin,
-    },
+    id: '',
+    name: '',
+    type: 'acscsEmail',
+    labelDefault: '',
+    labelKey: '',
+    uiEndpoint: window.location.origin,
 };
 
 function EmailIntegrationForm({
     initialValues = null,
     isEditable = false,
 }: IntegrationFormProps<ACSCSEmailIntegration>): ReactElement {
-    const formInitialValues = { ...defaultValues, ...initialValues };
+    let formInitialValues = { ...defaultValues, ...initialValues };
     if (initialValues) {
-        formInitialValues.notifier = {
-            ...formInitialValues.notifier,
+        formInitialValues = {
+            ...formInitialValues,
```

Comment:
This looks similar to another notifier that does not have stored credentials:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/SyslogIntegrationForm.tsx#L60-L86

For my understanding, where do callback functions from `useIntegrationForm` hook deal with presence or absence of `notifiers` property?

Reply:
Aha, is it an implicit expectation of the endpoint depending on stored credentials or not?

If so, what do you think about a comment to explain?

Reply:
Yes exactly, we're not storing credentials for this notifier in the backend. Instead we load the kubernetes service account during runtime as credentials.